webapp/controller/App.controller.js

```diff
@@ -9,12 +9,12 @@ sap.ui.define([
 	return Controller.extend('todo.controller.App', {
 
 		/**
-		 * adds a new to-do item using property <code>newTodo</code>
+		 * adds a new to-do item to the list.
```

Comment:
could you update the comments to "Adds a new to-do item to the end of the list."